examples/reminders/lib/feature_facebook_authentication/views/facebook_login_page.dart

```diff
@@ -66,7 +70,20 @@ class _FacebookLoginPageState extends State<FacebookLoginPage> {
             color: Colors.blue,
             onPressed: () =>
                 context.read<FirebaseBlocType>().events.logIn(anonymous: false),
-          )
+          ),
+          const SizedBox(
+            height: 20,
+          ),
+          RxBlocBuilder<FirebaseBlocType, bool>(
+            state: (bloc) => bloc.states.isLoading,
+            builder: (context, isLoading, bloc) {
+              if (isLoading.hasData && isLoading.data == true) {
+                return const AppProgressIndicator();
```

Comment:
Please show the loading indicator as part of the button instead of showing an indicator at the bottom of the page. If you are not sure how to achieve this. Please check out the loadingWithTag property as well as the provided builders

Reply:
I changed the loading indicator to be displayed instead of the text in the corresponding button and changed the loading state in the bloc to be:
`Stream<LoadingWithTag> get isLoading;`